pandas/core/reshape/reshape.py

```diff
@@ -751,6 +751,138 @@ def _convert_level_number(level_num, columns):
     return result
 
 
+def from_dummies(data, columns=None, prefix_sep="_", dtype="category", fill_first=None):
+    """
+    The inverse transformation of ``pandas.get_dummies``.
+
+    Parameters
+    ----------
+    data : DataFrame
+    columns : list-like, default None
```

Comment:
I would also rather not provide `columns` as a keyword; duplicative of the first argument

Reply:
Or is this supposed to be the inverse of the `prefix` argument in `get_dummies`? If so, I think should just reuse that name

Reply:
Sure, I'll change it to `prefix`